Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.yml

```diff
@@ -1,7 +1,4 @@
 category: Authentication & Identity Management
-sectionOrder:
-- Connect
-- Collect
```

Comment:
revert this part

Reply:
The pack was submitted via Xsoar Dev Web instance so I don't see this option in the yml file downloaded directly form the Dev WebGUI. 
I would agree. Please feel free to go ahead.